src/data-table/src/styles/index.cssr.ts

```diff
@@ -303,6 +303,9 @@ export default c([
     cM('loading', [
       cB('data-table-wrapper', 'opacity: var(--opacity-loading);')
     ]),
+    cM('striped', [
+      cB('data-table-tr:nth-child(even)', [cB('data-table-td', 'background-color: var(--td-color-hover);')])
+    ]),
```

Comment:
In fact this is not the correct behavior since when table row is expandable, the even odd status of the row will be changed.

However it won't be hard since we can just follow the index to distinguish the row status.